help/commands.md

```diff
@@ -1171,10 +1171,18 @@ Frequency distribution for *variable1*
 
  Add cumulative percentage
 
-- `!r`
+- `!pr`
 
  Add row percentage
 
+- `!ci`
+
+ Calculate confidence intervals for row percentage
+
+- `!w:=weightVariable`
```

Comment:
Does it matter that there are no spaces between the "w" and ":=" and "weightVariable"?

Reply:
I usually don't put in the spaces when I'm typing commands. We could show either in the help file and I suppose we should be consistent. Something for another day...